.github/workflows/Dockerfile.stack-deps

```diff
@@ -11,13 +11,17 @@ RUN    apt-get update        \
 
 RUN curl -sSL https://get.haskellstack.org/ | sh
 
+# The image is built specifically for an environment with this user/group
+ARG USER
+RUN echo "User: ${USER?'USER argument must be provided during docker build'}"
+ARG GROUP=$USER
 ARG USER_ID=1000
```

Comment:
Why not set this to USER_ID? from the build arg line?

Reply:
This is where USER_ID is defined as an argument. Saying `USER_ID=1000` is providing a default, but if you pass `--build-arg USER_ID=...` it overrides that default.